blog.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name= "viewport" content="width=device-width, initial-scale=1">
+    <title>Adopt a Dog - Home</title>
+  </head>
+  <body>
+    <header>
+      <div>
+        <img src="images/adoptadog-logo.svg" alt="Adopt a Dog Logo">
+        <p>Seattle's premiere dog adoption agency</p>
+      </div>
+      <div>
+        <img width="30" src="images/cart-icon.svg" alt="Shopping cart icon">
```

Comment:
make sure to pull out the width from your HTML. All styling should be handled by your css file